dev.k8slens.OpenLens.yml

```diff
@@ -38,6 +38,30 @@ modules:
           project-id: 5097
           url-template: https://github.com/libvips/libvips/archive/refs/tags/v$version.tar.gz
 
+  - name: kubelogin
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 bin/*/kubelogin "${FLATPAK_DEST}/bin/kubelogin"
```

Comment:
Just testing this on my local machine and discovered this line should be:
`install -Dm755 ./*/kubelogin "${FLATPAK_DEST}/bin/kubelogin"`

With that the build succeeds, I will now test to see if kubelogin can be found by the app.

Reply:
It works, `kubelogin` can now be found and I can reach my clusters.